src/test/java/io/appium/java_client/android/GetSessionDetailsTest.java

```diff
@@ -0,0 +1,58 @@
+package io.appium.java_client.android;
+
+import static org.junit.Assert.assertNotNull;
+
+import io.appium.java_client.remote.MobileCapabilityType;
+import io.appium.java_client.service.local.AppiumDriverLocalService;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import java.io.File;
+
+/**
+ * Created by saikrisv on 03/06/16.
+ */
+public class GetSessionDetailsTest {
+    public static AppiumDriverLocalService service;
+    protected static AndroidDriver<AndroidElement> driver;
+
+    /**
+     * initialization.
+     */
+    @BeforeClass public static void beforeClass() throws Exception {
+        service = AppiumDriverLocalService.buildDefaultService();
+        service.start();
+
+        if (service == null || !service.isRunning()) {
+            throw new RuntimeException("An appium server node is not started!");
+        }
+
+        File appDir = new File("src/test/java/io/appium/java_client");
+        File app = new File(appDir, "IntentExample.apk");
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+        capabilities.setCapability(MobileCapabilityType.DEVICE_NAME, "Android Emulator");
+        capabilities.setCapability(MobileCapabilityType.APP, app.getAbsolutePath());
+        driver = new AndroidDriver<>(service.getUrl(), capabilities);
+    }
+
+    /**
+     * finishing.
+     */
+    @AfterClass public static void afterClass() {
+        if (driver != null) {
+            driver.quit();
+        }
+        if (service != null) {
+            service.stop();
+        }
+    }
+
+
+    @Test public void getDeviceUDIDTest() {
```

Comment:
Why did you add this test class? Maybe it makes sense to add this method to https://github.com/appium/java-client/blob/master/src/test/java/io/appium/java_client/android/AndroidDriverTest.java. What do you think?